api/src/main/java/uk/gov/hmcts/reform/divorce/caseapi/controllers/CcdCallbackController.java

```diff
@@ -0,0 +1,46 @@
+package uk.gov.hmcts.reform.divorce.caseapi.controllers;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import io.swagger.annotations.ApiOperation;
+import io.swagger.annotations.ApiResponse;
+import io.swagger.annotations.ApiResponses;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RestController;
+import uk.gov.hmcts.reform.ccd.client.model.CallbackResponse;
+import uk.gov.hmcts.reform.divorce.caseapi.model.ccd.CcdCallbackRequest;
+import uk.gov.hmcts.reform.divorce.caseapi.notification.handler.SaveAndSignOutNotificationHandler;
+import uk.gov.hmcts.reform.divorce.ccd.model.CaseData;
+
+import java.util.Map;
+
+import static org.springframework.http.MediaType.APPLICATION_JSON_VALUE;
+
+@Slf4j
+@RestController
+public class CcdCallbackController {
```

Comment:
Please follow the convention I set out in https://github.com/hmcts/nfdiv-case-api/pull/25/files - the controller should be for the event and the method should be the callback type